Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN apt-get update \
 
 RUN pip install \
   pipenv
+  Pillow
```

Comment:
Missing a `\` on the above line, Docker won't build the image without it.

Reply:
It might be more than this, since adding the slash here and rebuilding the container doesn't work either. An error is thrown during the `Step 8/14 : RUN pipenv install --system --dev` step. Looks like the actual exception is `ModuleNotFoundError: No module named 'numpy'"`, it looks related to installing libtiff.

Reply:
Perhaps the better question is how are you running the tests? Did you build a virtualenv and the whole thing that the docker container is trying to do?

Reply:
I use pipenv locally. When I first inherited the project, Amy and I tried to get the Docker container up and running for about half a day, but then moved on.